docs-src/generate.js

```diff
@@ -43,7 +43,13 @@ function genJS(items) {
   browserify(jsSourceFile, {debug: true})
     .transform('babelify')
     .bundle()
-    .on('end', () => fs.unlinkSync(jsSourceFile))
+    .on('end', () => {
+      try {
+        fs.unlinkSync(jsSourceFile)
+      } catch(err) {
+        console.log(err);
```

Comment:
Did a drive by on this because unlinkSync kept killing the dev server and was logging no errors